docs/customise-client/requests.rst

```diff
@@ -51,4 +51,4 @@ If a map is displayed by Auspice the individual tiles are requested from api.map
 Google Analytics (Optional)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Auspice has the potential to include Google Analytics in a limited fashion. See `the client customisation API documentation <customise-client/api.md#available-customisations>`__ for how to set this. Note that these requests are only made if you specify a Google Analytics key during build-time customisation of the client.
+Auspice has the potential to include Google Analytics in a limited fashion. See :ref:`the client customisation API documentation <client-api-available-customisations>` for how to set this. Note that these requests are only made if you specify a Google Analytics key during build-time customisation of the client.
```

Comment:
[Changes required] The analytics content has changed (in the original markdown file) since this branch was created so this section needs updating - see https://github.com/nextstrain/auspice/pull/1597/files

Reply:
This is based on f06ff45db912a8acba4a0828d02dd49468dd454e so the changes are already there on the client API page ([preview](https://nextstrain--1588.org.readthedocs.build/projects/auspice/en/1588/customise-client/api.html)) ([src](https://github.com/nextstrain/auspice/blob/f0fda9dd7e139f8a7b12ee1a444104f8288e6873/docs/customise-client/api.rst?plain=1#L172-L179)). Am I missing something?

Reply:
Oh we're both missing something! This comment is on the _requests_ page, which we seem to have forgotten to update during #1597. Would you mind updating it as part of this PR? Suggested text:

> Auspice has the potential to include either Plausible Analytics or Google Analytics in a limited fashion. See :ref:`the client customisation API documentation <client-api-available-customisations>` for how to set this. Note that these requests are only made if you set it up build-time customisation of the client. (For nextstrain.org we use [Plausible analytics](https://github.com/nextstrain/nextstrain.org/blob/master/auspice-client/customisations/config.json#L14).)

(heading needs changing too)

Reply:
Done: 653f4403211b2e038b2084f1e7244fe1bc143ac9